.circleci/config.yml

```diff
@@ -5,7 +5,7 @@ jobs:
     docker:
       - image: circleci/android:api-28-alpha
     environment:
-      _JAVA_OPTIONS: -Xmx3600m
+      _JAVA_OPTIONS: -Xmx2048m -Dorg.gradle.daemon=false
```

Comment:
Why decrease it?

Reply:
I tried dividing the workers into 4 with 3600 and it was still giving OOM error, so I tried decreasing the allowed memory and it worked

Reply:
Let's hope it continues to